style.css

```diff
@@ -619,32 +620,595 @@ footer {
   margin-left: 0;
 }
 
-/*
-@media screen and (min-width: 48rem) {
-  header {
-    width: 96%;
+@media screen and (max-width: 425px) and (min-width: 320px) {
+  .profession--desktop,
+  .desktop--nav,
+  .image24,
+  .image25,
+  .desktop--main {
+    display: none;
   }
 
-  .whoiam {
+  .desktop--projectss {
+    display: none;
+  }
+
+  .Aboutme--desktop {
+    display: none;
+  }
+
+  .desktop--form,
+  .contactme--desktop-img1 {
+    display: none;
+  }
+
+  .desktop--form--1,
+  .contactme--desktop-img2 {
+    display: none;
+  }
+
+  .desktop--para {
+    display: none;
+  }
+
+  .cv-desktop {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 48rem)  and (max-width: 90rem) {
```

Comment:
- [ ] please, kindly use PX for the media query. PX is an absolute measurement, and it should be used for media queries. Whereas,  rem are relative measurements, and are good for space/font size settings. . 
- the max can be removed as there are larger screen sizes and since your work is responsive it will fit in them
this will do ...`@media screen and (min-width: 768px)`